doc/install-autocrypt-enigmail.rst

```diff
@@ -16,7 +16,6 @@ You can test Autocrypt in Enigmail already today:
 
 - The following setup may require a restart of Thunderbird
 
-- Then, navigate to "Edit / Preferences / Advanced / Config Editor", read the warning and 
-  scroll down to "extensions.enigmail.autocryptMode": Double-click the entry and enter a `1`
+- Autocrypt is now running. You can configure it at: Account Manager > OpenPGP Security.
```

Comment:
"is now running" sounds odd.  Shouldn't it just say "You can enable autocrypt here: Account Manager > OpenPGP Security" and maybe even mention the precise option? (Haven't installed the latest nightly yet so can't say what it is)

Reply:
It's more disable, right?
How about "Autocrypt is part of the default configuration. You can configure and disable it at:..." (If one can also configure it...)
Will install latest nightly tomorrow and try.

Reply:
I agree with @azul, that sounds better.

Reply:
I'm fine with "Autocrypt is part of the default configuration", but we should remove the bullet point - it isn't part of the step-by-step guide anymore.